ui/components/ui/form-field/form-field.js

```diff
@@ -210,6 +212,13 @@ FormField.propTypes = {
   titleDetailWrapperProps: PropTypes.shape({
     ...Box.propTypes,
   }),
+  /**
+   * Props to pass to wrapping Box component of the titleHeading component
+   * Accepts all props of the Box component
+   */
+  titleHeadingWrapperProps: PropTypes.shape({
+    ...Box.propTypes,
+  }),
```

Comment:
Adding `titleHeadingWrapperProps` as the fallback prop to add props to `Box` here. cc @georgewrmarshall